release_notes.md

```diff
@@ -4,6 +4,35 @@ title: Release Notes
 permalink: /release_notes/
 ---
 
+## 1.9.0.0.7062 - October 14th 2017
+
+* Add option to securely save lobby credentials (#2395).
+* Fix console opening when playing a network game (#2289).
+* Fix threading errors reported when starting Map Creator (#2238).
+* Improve security of network game authentication (#2107).
+* Add option to securely save PBEM/PBF credentials (#1955).
+* Fix out-of-date map check (#1695).
+* Fight a single battle automatically
+* Upgrade retreat options - no double prompt with one valid territory
+  * don't offer prompt when only defenseless units remain
+* Show dice details by default in PBF/PBEM
+* Auto fight battles with no defenders who can fire and no dependent battles
+* Add experimental savegame format (#2128)
```

Comment:
I'm going to suggest removing this line.  Since it's an optional beta feature, disabled by default, and not yet fully implemented, I don't think it will be meaningful to the average user.

Reply:
ok